src/sentry/middleware/flag.py

```diff
@@ -0,0 +1,15 @@
+from collections.abc import Callable
+
+from django.http.request import HttpRequest
+from django.http.response import HttpResponseBase
+
+from sentry.utils.flag import initialize_flag_manager
+
+
+class FlagMiddleware:
```

Comment:
should this class extend something, or maybe it would be easier to make it a func:

```
def access_log_middleware(
    get_response: Callable[[Request], Response]
) -> Callable[[Request], Response]:
    def middleware(request: Request) -> Response:
        initialize_flag_manager()
        return get_response(request)
```